CRISPResso2/CRISPRessoAggregateCORE.py

```diff
@@ -62,6 +62,7 @@ def main():
         parser.add_argument('--place_report_in_output_folder',  help='If true, report will be written inside the CRISPResso output folder. By default, the report will be written one directory up from the report output.', action='store_true')
         parser.add_argument('--suppress_report',  help='Suppress output report', action='store_true')
         parser.add_argument('--suppress_plots',  help='Suppress output plots', action='store_true')
+        parser.add_argument('--max_samples_per_summary_plot', type=int, help="Maximum number of samples on each page of the pdf report plots. If this number gets above ~150, they will be too big for matplotlib.", default=250)
```

Comment:
@kclem did we want to have this default to 150? I noticed that 150 is in the description, but then the default is 250.